### DISCUSSION TITLE
Ability to modify widgets

### DISCUSSION THREAD

**Discussion by jmginfor** [YetiForceCompany/YetiForceCRM#15096] [Feature improvement]
It will be useful if somehow we can easily modify widgets. 

Exemple : 
I want monthly total sale amount VS previous years. 

I can create a widget to compare August 2020 VS August 2021, but next years I'll need to erase and remake the widget instead of just clicking on modify widgets and adds a new year to it.

Will be useful! Thanks

**Comment by mariuszkrzaczkowski** [comment]
In the development version there is such a possibility, it has recently been added :)
https://gitdeveloper.yetiforce.com/index.php?module=WidgetsManagement&parent=Settings&view=Configuration

![image](https://user-images.githubusercontent.com/10198654/130397023-82d3b787-2c50-474d-b6f7-f67620489884.png)
![image](https://user-images.githubusercontent.com/10198654/130397036-57f88075-86f6-4d90-93de-b06361837bc6.png)
![image](https://user-images.githubusercontent.com/10198654/130397053-80cef14e-4664-4f57-bf4c-be3224a1d911.png)

**Reply by jmginfor** [reply]
Ho yes! Thanks for your reply.
